File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.js
function solution(my_string) {
    let el = my_string.split(" ");
    let answer = Number(el[0]);
    
    el.forEach((item, idx) => {
        if(item === "+") {
            answer += Number(el[idx + 1]);
        }
        if(item === "-") {
            answer -= Number(el[idx + 1]);
        }
    })
     
    return answer;
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] s = my_string.trim().split(" ");
        int oper = 1;
        for (String s1 : s) {
            if(s1.equals("+") || s1.equals("-")){
                oper = s1.equals("-") ? -1 : 1;
            }else{
                answer += Integer.parseInt(s1) * oper;
            }
        }
        return answer;
    }
}